index.js

```diff
@@ -0,0 +1,82 @@
+
+window.onload = function(){
+    const app = document.querySelector('#app');
+
+    if (sessionStorage.getItem('LoggedIn')) {
+        loadPage('evaluations') // show evaluations page if the user is logged
+    } else {
+        loadPage('login') // if is not logged go to login page
+    } 
+}
+
+const loadPage = function(page) {
+    
+    switch (page) {
+        case "login":
+        app.innerHTML = LoginPage(); // call LoginPage function
+        loginEvents() // call LoginPage events function
+            break;
+
+        case "evaluations":
+        app.innerHTML = EvaluationsPage();
+        evaluationEvents() // call evaluation Page events function
+        commonEvents() // call commonEvents function which will load the login page
+            break;
+
+        case "new-evaluation":
+        app.innerHTML = NewEvaluationPage();
+        newEvaluationEvents() // call newEvaluation events function
+        commonEvents() // call commonEvents function which will load the login page
+            break;
+    }
+}
+
+interviewApp =  {}
+
+
+(function() {
+    var _setupEvents = function() {
+        loginEvents(),
```

Comment:
I imagine these methods are still pending updates? I'd expect that each module would have its own rendering and event setup, neatly separated away into its own sub-namespace. In our current situation, it doesn't really make sense to render / setup the login, evaluations and new evaluation modules at the same time, since they should be mutually exclusive modules, thus would never coexist, the way I see it.

Reply:
I understand, anyway I changed the structure there. :)